src/java/org/apache/cassandra/service/accord/txn/TxnRead.java

```diff
@@ -67,6 +73,12 @@ public TxnRead(List<TxnNamedRead> items, Keys txnKeys)
         this.txnKeys = txnKeys;
     }
 
+    public static TxnRead createRead(SinglePartitionReadCommand readCommand)
```

Comment:
nit/naming/non-binding: I'd be fine w/ `create()` too, given it's a static factory

Reply:
How about `createSerialRead` because this specifically creates one for that purpose?